nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/pom.xml

```diff
@@ -85,6 +85,25 @@
             <artifactId>nifi-web-servlet-shared</artifactId>
             <version>2.6.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-deploy</artifactId>
```

Comment:
The `jetty-deploy` library brings in several other dependencies. Is there a reason this is necessary in Jetty 12.1 and not 12.0?

Reply:
Yes, this is one of the changes with Jetty 12.1: `DeploymentManager` no longer exists and I'm replacing it by `StandardDeployer` which comes from `jetty-deploy` in `JettyServer`